.github/workflows/ci-build.yml

```diff
@@ -151,7 +151,7 @@ jobs:
         with:
           repository: ral-facilities/datagateway-api
           path: datagateway-api
-          ref: v6.0.0
+          ref: feature/icat-backend-generator#369
```

Comment:
I know you know but these references will need to change before merging this PR

Reply:
These references have been updated in the latest commit.